public/javascript/main.js

```diff
@@ -376,5 +376,21 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $("a.download-with-term").on("click", function(e){
```

Comment:
Such a minor thing, but not sure you need a the `a` in `$("a.download-with-term")`